### PR TITLE
Add arithmetic analysis test suite

### PR DESCRIPTION
**Pull request by larshum**
Adds a test suite for the `arithmetic_analysis` directory. Also fixes usage of global scope in files in that folder.

Note that the `newton_raphson_method` file is broken and it requires too much effort to fix. The tests for that method are failing because of this.

### CODE REVIEW

**Comment by larshum** [comment]
Fixes #55

File: tests/arithmetic_analysis/test_newton_raphson_method.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from math import sqrt
+import pytest
```
Comment (mnil2) [inline tests/arithmetic_analysis/test_newton_raphson_method.py]:
```suggestion
```
Importing from pytest twice

**Review by mnil2 — APPROVED** [review]
Approved :+1: Just found an unnecessary import

**Comment by larshum** [comment]
Tests are failing because of broken implementation (not our responsibility), so I'm merging now.

**Pull request merged**